doc/run/workflow_mode.md

```diff
@@ -0,0 +1,91 @@
+# Running SQLFlow as Workflow Mode
+
+This tutorial introduces how to setup the SQLFlow with workflow mode, you can
+
+1. Find why SQLFlow needs Workflow from [workflow](/doc/design/workflow.md); and
+1. How we implement it from [workflow package](/doc/design/workflow_package.md).
+
+The following figure shows a typical SQLFlow workflow architecture:
+
+![](figures/workflow.png)
+
+1. Users type in the SQL program on the client side, like Jupyter Notebook. the SQLFlow plugin on Notebook can send the SQL program to SQLFlow server via gRPC.
```

Comment:
Again, I don't see that we should NOT use bullets here, because there is no side-by-side logic here.  You might want to say

> The SQLFlow gRPC server accepts a SQL program as the request and translates it into a workflow.  Some steps of the workflow send normal SQL statement to the database system, and some others might submit an AI job to train a model, to predict using a trained model, or to visually explain a trained model.

Anyway, I don't think we need a figure here.